test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,77 @@
+require('dotenv').config();
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'Ktagudelo';
+const repositories = 'repos';
+const repositoryName = 'Proyecto-marvel';
+
+describe('GitHub User Create And Modify Issue With A Title', () => {
+  it('Consume GET GitHub User Logged', async () => {
+    const response = await axios.get(`${urlBase}/user`, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    if (response.data.public_repos > 1) {
+      expect(response.status).to.equal(StatusCodes.OK);
+    }
```

Comment:
you can rewrite this:
```suggestion
    expect(response.status).to.equal(StatusCodes.OK);
    expect(response.data.public_repos).to.be.above(0)
```